opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/EventRestService.java

```diff
@@ -67,6 +69,9 @@
     @Autowired
     private EventDao m_eventDao;
 
+    @Autowired
+    private EventIpcManager m_eventForwarder;
```

Comment:
I typically use EventProxy. Check how I did it for the NBIs:

https://github.com/OpenNMS/opennms/blob/foundation-2016/opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/config/SyslogNorthbounderConfigurationResource.java